web/src/tippyjs.js

```diff
@@ -413,6 +413,11 @@ export function initialize() {
             instance.destroy();
         },
     });
+
+    delegate("body", {
+        target: "#user_info_popover  .status-emoji",
```

Comment:
There's two spaces between these selectors. I'm surprised that wasn't linted out, but be sure to fix.

Reply:
Thanks @karlstolley for the valuable feedback. I have fixed the issue PTAL.